src/systemdhook.c

```diff
@@ -790,15 +790,6 @@ int main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
-	/* Extract values from the state json */
```

Comment:
should we keep this code here to check the root_path rather than at line 874 so it can fail earlier rather than doing all the other processing first?